modules/reference/pages/rpk/rpk-connect/rpk-connect.adoc

```diff
@@ -0,0 +1,85 @@
+= rpk connect
+
+A stream processor for mundane tasks.
+
+Either run Redpanda Connect as a stream processor or choose a command:
+
+Available commands:
+
+- xref:reference:rpk/rpk-connect/rpk-connect-echo.adoc[] - Parse a config file and echo back a normalized version.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-lint.adoc[] - Parse Redpanda Connect configs and report any linting errors.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-streams.adoc[] - Run Redpanda Connect in streams mode.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-list.adoc[] - List all Redpanda Connect component types.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-run.adoc[] - Run Redpanda Connect in normal mode against a specified config files.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-create.adoc[] - Create a new Redpanda Connect config.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-test.adoc[] - Execute Redpanda Connect unit tests.
+
+- `rpk connect template` - Interact and generate Redpanda Connect templates.
+
+- `rpk connect blobl` - Execute a Redpanda Connect mapping on documents consumed via stdin.
+
+- `rpk connect studio` - Interact with Benthos studio (https://studio.benthos.dev).
+
+- `rpk connect help` - Shows a list of commands or help for one command.
```

Comment:
Should these all be removed from this list for now?

Reply:
I mean the commands really exist, we just don't have reference for it.

Reply:
@vladoschreiner WDYT?

Reply:
I suggest removing the entire list. Available commands are listed in the left nav. And other top-level commands (e.g., [`rpk cluster`](https://deploy-preview-529--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster/)) don't list them on the page.

Reply:
I'll remove for now, but please @vladoschreiner take a look when you can.